oat-sa/tao-test-runner-qti-fe
# Feature/tca 307/review panel expose status

Related to : https://oat-sa.atlassian.net/browse/TCA-307
 
TestRunner - ReviewPanel - Expose the status of parts, sections and items.
  
#### How to test

- Create a test with the review panel enabled
- Start a test as a test taker
- Navigate to the review panel via Tab
- Check that screen reader announces:
  - the label of the test part (if present)
  - the label of the test section (if present)
  - the position of the focused item
  - the state of the focused item (answered, viewed, etc.)
  - that it is a link
  - that it is current
  - the label of the item

(Keyboard navigation issues are not taken into account here)
